config/machine/linux_clang_x86_64.mk

```diff
@@ -18,11 +18,11 @@ include config/extra/with-threads.mk
 
 CPPFLAGS+=-march=haswell -mtune=skylake
 CPPFLAGS+=-DFD_HAS_INT128=1 -DFD_HAS_DOUBLE=1 -DFD_HAS_ALLOCA=1 -DFD_HAS_X86=1 -DFD_HAS_SSE=1 -DFD_HAS_AVX=1
+LDFLAGS+=-lstdc++
```

Comment:
Is there a way we can add this only when doing asan?   I know you will get pushback adding to the dependency list here...